Content.Server/Chat/Systems/SpeakOnTriggerSystem.cs

```diff
@@ -0,0 +1,42 @@
+using Content.Shared.Timing;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Content.Server.Explosion.EntitySystems;
```

Comment:
Someone didn't sort their imports...

```suggestion
using Content.Server.Explosion.EntitySystems;
using Content.Shared.Timing;
using Robust.Shared.Prototypes;
using Robust.Shared.Random;
```

Reply:
Rider is to blame for this folly. I let it do its thing and it decided not to...